DigitalLearningSolutions.Web/Helpers/FilteringHelper.cs

```diff
@@ -9,13 +9,29 @@
     public static class FilteringHelper
     {
         public const char Separator = '|';
+        public const char FilterSeparator = '╡';
+
+        public static string? AddNewFilterToFilterBy(string? filterBy, string? newFilterValue)

```

Comment:
filterBy hasn't been renamed to currentFilters

Reply:
The filter string is called filterBy, including in the method name. Either we need to change it to currentFilters absolutely everywhere (and maybe update sortBy to match) or we keep it as filterBy here so I went with the easier option.

Reply:
Fair point. In which case, I agree that filterBy should be retained as the name here.